sql/core/src/test/scala/org/apache/spark/sql/execution/joins/BroadcastJoinSuite.scala

```diff
@@ -85,31 +86,39 @@ class BroadcastJoinSuite extends QueryTest with SQLTestUtils {
     plan
   }
 
+  // This tests here are failed on Windows due to the failure of initiating executors
+  // by the path length limitation. See SPARK-18718.
```

Comment:
The same instance with https://github.com/apache/spark/pull/16147#discussion_r90870969. Initiating executor as process is continuously being failed due the path length limitation on Windows. This makes proceeding tests on Windows harder.

See the built command to launch executor as process here - https://gist.github.com/HyukjinKwon/5bc81061c250d4af5a180869b59d42ea